prisma/schema.prisma

```diff
@@ -7,5 +7,24 @@ generator client {
   provider = "prisma-client-js"
 }
 
+model Event {
+  id            Int       @default(autoincrement()) @id
+  title         String
+  description   String?
+  location      String
+  startTime     DateTime
+  endTime       DateTime
+  createdBy     User?       @relation(fields: [userId], references: [id])
```

Comment:
can you call it `owner`? also external guests can create events. the owner will always be the owner of the event, the external guest that creates the event can also be a participant

Reply:
Changed